Mathlib/Algebra/RingQuot.lean

```diff
@@ -73,10 +73,11 @@ theorem Rel.sub_right {R : Type uR} [Ring R] {r : R → R → Prop} ⦃a b c : R
 theorem Rel.smul {r : A → A → Prop} (k : S) ⦃a b : A⦄ (h : Rel r a b) : Rel r (k • a) (k • b) := by
   simp only [Algebra.smul_def, Rel.mul_right h]
 
+open Relation in
 /-- `EqvGen (RingQuot.Rel r)` is a ring congruence. -/
 def ringCon (r : R → R → Prop) : RingCon R where
   r := EqvGen (Rel r)
-  iseqv := EqvGen.is_equivalence _
+  iseqv := Relation.EqvGen.is_equivalence _
```

Comment:
Do you need both of these changes?

Reply:
The alternative would be to add `Relation.` namespaces everywhere in the declaration.

Reply:
@semorrison I've gone ahead and removed the extraneous `open Relation`s.